FreeRTOS/iot-reference-arm-corstone3xx
# Remove vht support

Description
-----------
In the initial days of FRI, vht running in ami was the only supported platform since it supported virtual socket interface. However, latest ecosystem FVPs do support virtual socket interface. In addition, FRI now supports FreeRTOS TCP/IP stack. Therefore, remove vht support.

Test Steps
-----------
Changes validated in the CI, no regression exists.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
